### DISCUSSION TITLE
Normals visualization in gui::Window app

### DISCUSSION THREAD

**Discussion by vovo-4K** [isl-org/Open3D#4211] [Q&A]
I found several concurrent ways to create GUI applications using Open3D.
Simple Visualizer has the ability to set RenderOptions to enable normals visualization.
But how to enable normals rendering on gui::SceneWidget?

Upd. I'm using C++ API. C++ documentation is not user friendly a bit

**Comment by vovo-4K** [comment]
Ok. I found a workaround with the manual creation of geometry::LineSet and adding it as a separate Geometry entity to the scene.
But is there any more elegant way to visualize normals from geometry::PointCloud normals_ field?

**Comment by ssheorey** [comment]
Hi @vovo-4K we are migrating Open3D visualization to a new backend, with support for creating GUIs. The new functionality is in the `gui:: `and `rendering::` namespaces. As such some features (such as the hedgehog normals visualization) are not available yet. Please open a feature request.
